ql/src/java/org/apache/hadoop/hive/ql/QueryInfo.java

```diff
@@ -27,9 +31,10 @@ public class QueryInfo {
   private final long beginTime;
   private final String sessionId;
   private final String operationId;
+  private final Instant beginTimeUTC;
```

Comment:
why do we need time vars for several TZ?

Reply:
we need this in query history where we store start/end time in server's timezone and in utc for convenience reasons, but I can achieve it without storing it in a field, simply:
```
  public Instant getBeginTimeUTC() {
    return Instant.ofEpochMilli(beginTime);
  }
```
this way maybe I can get away without the jackson dependency too